openpathsampling/analysis/tis_analysis.py

```diff
@@ -353,6 +343,9 @@ def total_crossing_probability(self, storage=None, method="wham", force=False):
         force : bool (False)
             if true, cached results are overwritten
         """
+
+        tcp = dict()
```

Comment:
Okay, my mistake. Indeed the autocomplete is for the self.tcp which would be good to exists from the start. I misunderstood. The tcp = dict() was only meant to make sure that the last line does not break. I was not aware that this is what you wanted.